.gitignore

```diff
@@ -7,3 +7,5 @@ release-announcement
 .project
 .vscode
 cluster/.*
+vendor/**
```

Comment:
A few other PRs have added this line and it's caused dep issues for devs.  Is there a reason for excluding new vendored dependencies?

Reply:
Good catch!  This was a merge/rebase issue.  I only intended to add .history.